src/main/java/fr/esgi/al5_2/Tayarim/controllers/AdministrateurController.java

```diff
@@ -0,0 +1,173 @@
+package fr.esgi.al5_2.Tayarim.controllers;
+
+import fr.esgi.al5_2.Tayarim.dto.proprietaire.*;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.AvoidStarImportCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Using the '.*' form of import should be avoided - fr.esgi.al5_2.Tayarim.dto.proprietaire.*.